backend/qemu.pm

```diff
@@ -275,10 +275,19 @@ sub start_qemu {
         die "no dmi data for '$vars->{LAPTOP}'\n" unless -d "$bmwqemu::scriptdir/dmidata/$vars->{LAPTOP}";
     }
 
+    my $bootfrom = "";    # branch by "disk" or "cdrom", not "c" or "d"
     if ($vars->{BOOT_HDD_IMAGE}) {
         # skip dvd boot menu and boot directly from hdd
         $vars->{BOOTFROM} //= 'c';
     }
+    if ($vars->{BOOTFROM} eq "d" || $vars->{BOOTFROM} eq "cdrom") {
```

Comment:
This causes warnings when `BOOTFROM` variable is not defined at all. Please fix.